### PR TITLE
Multi-VO fts cronjob

### PR DESCRIPTION
**Pull request by gmatthews20**
Add documentation for the multi-vo fts cronjob

rucio/containers#226
rucio/helm-charts#126

### CODE REVIEW

File: docs/multi_vo_rucio.md [hunk 1]
```diff
@@ -62,6 +62,56 @@ not be set for the server or the daemons as these parts of Rucio are not
...
+can submit jobs to FTS. For the CA you have to add a
+`<releasename>-rucio-ca-bundle` secret. For the user proxy a cronjob can be
+setup to either generate it from a long proxy. The cronjob uses the
+[fts-cron](https://github.com/rucio/containers/tree/master/fts-cron) container
+which expects different input secrets. When enabled, the cronjob runs once upon
+installation and then every 6 hours. 
+
+For multi-vo the helm chart needs some changes vo should be set to `multi_vo`
+and a `vomses` section added. This section contains the vo and voms extention
+if required. Any additiional VOMS server info can be mounted as an additional
```
Comment (Thysk) [inline docs/multi_vo_rucio.md:77]: `Any additiional VOMS server`
Should be `Any additional VOMS server`

File: docs/multi_vo_rucio.md [hunk 2]
```diff
@@ -62,6 +62,56 @@ not be set for the server or the daemons as these parts of Rucio are not
 associated with a single VO. If `multi_vo` is not set, or set to False, then
 Rucio will operate normally.
 
+### Renew FTS cronjob
+
+Some daemons need a delegated X509 user proxy and the necessary CA so that it
+can submit jobs to FTS. For the CA you have to add a
+`<releasename>-rucio-ca-bundle` secret. For the user proxy a cronjob can be
+setup to either generate it from a long proxy. The cronjob uses the
```
Comment (Thysk) [inline docs/multi_vo_rucio.md:70]: `either generate it from a long proxy`
suggests two methods, but only one described

**Review by Thysk — COMMENTED** [review]
Some typos but otherwise I think is clear

File: docs/multi_vo_rucio.md (hunk 1 above)
Comment (gmatthews20) [reply docs/multi_vo_rucio.md]: Fixed

File: docs/multi_vo_rucio.md (hunk 2 above)
Comment (gmatthews20) [reply docs/multi_vo_rucio.md]: Fixed

**Comment by bari12** [comment]
This needs to be rebased. The files all moved slightly :-)

**Pull request closed without merging**